data/benefits/dynamic/fsl.ts

```diff
@@ -539,38 +397,11 @@ function formatBenefit(customizationBenefit, institution) {
   }
 }
 
-export function getInstitutionName(institutionsMap, code) {
-  // D = département ; M = métropole
-  const geographicalEntity = code[0]
-  // code Insee pour le département ou code Siren pour la métropole
-  const geographicalCode = code.slice(1)
-  let institutionName
-
-  if (geographicalEntity === "D") {
-    institutionName = getDepartmentInstitutionByInseeCode(
-      institutionsMap,
-      geographicalCode
-    )
-  } else {
-    institutionName = getMetropoleInstitutionBySirenCode(
-      institutionsMap,
-      geographicalCode
-    )
-  }
-  return institutionName
-}
-
-export function build(institutionsMap) {
-  const result: any = Object.keys(FSL_BY_CODE).reduce(
-    (accum: any, code: string) => {
-      const customizationBenefit = FSL_BY_CODE[code]
-      const institutionName = getInstitutionName(institutionsMap, code)
-
-      if (!institutionName) {
-        console.warn(`No instistution for metropole fsl ${code}`)
-        return accum
-      }
-      const benefit = formatBenefit(customizationBenefit, institutionName)
+export function build() {
+  const result: any = Object.keys(FSL_BY_INSTITUTION_SLUG).reduce(
+    (accum: any, institutionId: string) => {
+      const customizationBenefit = FSL_BY_INSTITUTION_SLUG[institutionId]
+      const benefit = formatBenefit(customizationBenefit, institutionId)
```

Comment:
```suggestion
  const result: any = Object.entries(FSL_BY_INSTITUTION_SLUG).reduce(
    (accum: any, [institutionId, InstitutionData]) => {
      const benefit = formatBenefit(institutionData, institutionId)
```

`entries` permet d'éviter d'avoir a récupérer les data a partir des clefs

Reply:
Je pense que ta proposition ne vient pas résoudre le point souligné par @Cugniere mais il me corrigera si je me trompe. Dans ta proposition, il y a toujours l'incohérence `FSL_BY_INSTITUTION_SLUG` / `institutionId`.

Reply:
j'ai renommé la clef mais il n'empêche que ça fait une opération en moins